claim/models.py

```diff
@@ -1,17 +1,92 @@
 import datetime
 import json
 
+from django.db.models import Q
 from django.db import models
 from django.contrib.auth.models import User
-from django.conf import settings
 from django.db.models import Count
 from django.utils.translation import ugettext as _
 
 
+class OrganizationType(models.Model):
+    # TODO(vegasq) should we use AMENITY as keys here?
+    ORG_TYPES = (
+        ("0", _("Unknown")),
+        ("1", _("Міністерство фінансів України")),
+        ("2", _("Міністерство соціальної політики України")),
+        ("3", _("Міністерство регіонального розвитку, "
+                "будівництва та житлово-комунального господарства України")),
+        ("4", _("Міністерство охорони здоров'я України ")),
+        ("5", _("Міністерство освіти і науки України")),
+        ("6", _("Міністерство оборони України")),
+        ("7", _("Міністерство молоді та спорту України")),
+        ("8", _("Міністерство культури України")),
+        ("9", _("Міністерство інфраструктури України")),
+        ("10", _("Міністерство інформаційної політики України")),
+        ("11", _("Міністерство закордонних справ України")),
+        ("12", _("Міністерство енергетики та вугільної промисловості "
+                 "України")),
+        ("13", _("Міністерство економічного розвитку і торгівлі України")),
+        ("14", _("Міністерство екології та природних ресурсів України")),
+        ("15", _("Міністерство внутрішніх справ України")),
+        ("16", _("Міністерство аграрної політики та продовольства України")),
+        ("17", _("Міністерство юстиції України")),
+    )
+    org_type = models.CharField(choices=ORG_TYPES,
```

Comment:
This geojson is raw export from OSM. So the question about parameters in it is still open. Here is doc with lazy discussion on it on FB: https://goo.gl/qtycPU. I left amenity in json because I saw some unique info in this field. If we'll encrypt organization type in ID field we don't need amenity at all.